pkg/cloud/cloud.go

```diff
@@ -331,12 +335,15 @@ func (c *cloud) CreateDisk(ctx context.Context, volumeName string, diskOptions *
 	clientToken := sha256.Sum256([]byte(volumeName))
 
 	requestInput := &ec2.CreateVolumeInput{
-		AvailabilityZone:  aws.String(zone),
-		ClientToken:       aws.String(hex.EncodeToString(clientToken[:])),
-		Size:              aws.Int64(capacityGiB),
-		VolumeType:        aws.String(createType),
-		TagSpecifications: []*ec2.TagSpecification{&tagSpec},
-		Encrypted:         aws.Bool(diskOptions.Encrypted),
+		AvailabilityZone: aws.String(zone),
+		ClientToken:      aws.String(hex.EncodeToString(clientToken[:])),
+		Size:             aws.Int64(capacityGiB),
+		VolumeType:       aws.String(createType),
+		Encrypted:        aws.Bool(diskOptions.Encrypted),
+	}
+
+	if !util.IsSBE(zone) {
```

Comment:
What's the CreateVolume behavior if tags are provided for SBE volumes?

Reply:
It fails with error that input is malformed.